src/library_webgl.js

```diff
@@ -376,7 +376,11 @@ var LibraryGL = {
       var source = '';
       for (var i = 0; i < count; ++i) {
         var len = length ? {{{ makeGetValue('length', 'i*4', 'i32') }}} : -1;
+#if MEMORY64
+        source += UTF8ToString({{{ makeGetValue('string', 'i*8', '*') }}}, len < 0 ? undefined : len);
+#else
         source += UTF8ToString({{{ makeGetValue('string', 'i*4', 'i32') }}}, len < 0 ? undefined : len);
```

Comment:
Can you just use `i*{POINTERSIZE}` and `*` here, which should work in both cases I think?

Reply:
thanks, will make that change.